products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,232 @@
+## Game of Thrones deaths
+
+This article is a part of a series intended to show how to use Memgraph
+on real-world data to retrieve some interesting and useful
+information.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials-overview.md).
+
+### Introduction
+
+**WARNING** - If you've been living under a rock and still have not watched Game of Thrones,
+this tutorial could contain ***spoilers***. 
+
+We can all agree that 2020. was one of the worst years in our lives because of the worldwide
+pandemic, 2019. still comes close with that disappointing, poorly written ending to one of the
+best shows of all time, Game of Thrones. Seven-year build-up to that awful last season crushed
+the souls of millions of fans around the world. Nevertheless, the Game of Thrones world is full
+of interesting characters, locations, scenarios, and most importantly deaths. If you want to know
+how many characters would survive if Jon Snow stayed dead, which House has the best Kill/Death Ratio
+or who was the biggest traitor in the show, you have come to the right place!
+
+### Data Model
+
+Although you know the Game of Thrones TV show is based on series of books called A Song of Ice and
+Fire, our graph database contains only characters from said TV shows as books are still not finished.
+This tutorial would not be possible with data analyst David Murphy who shared his collection of
+on-screen deaths on [this link](https://data.world/datasaurusrex/game-of-thones-deaths). Some of the rules you have to take into concern are that kills that
+happened off-screen and undead(wraiths) are not counted. The dataset we used was slightly modified, to 
+be precise, columns "Episode Name" and "IMDb Rating" were added.
+
+Now, the data we'll be using in our queries can be classified as follows:
+  * nodes, labeled as "Character", "Allegiance", "Death", "Episode", "Season", or "Location"

```

Comment:
You already have a list of node labels below so this is insufficient. Check here how it is done (https://docs.memgraph.com/memgraph/tutorials-overview/football-transfers#data-model) from point of structure, formatting edge types and node labels as well as short description and example.